api4/file_test.go

```diff
@@ -817,7 +818,11 @@ func TestGetFileHeaders(t *testing.T) {
 	t.Run("txt", testHeaders(data, "test.txt", "text/plain", false, false))
 	t.Run("html", testHeaders(data, "test.html", "text/plain", false, false))
 	t.Run("js", testHeaders(data, "test.js", "text/plain", false, false))
-	t.Run("go", testHeaders(data, "test.go", "application/octet-stream", false, false))
+	if runtime.GOOS == "linux" || runtime.GOOS == "darwin" {
```

Comment:
Maybe we can do this just for macos.
```suggestion
	if runtime.GOOS == "darwin" {
```

Reply:
It fails on my machine as well. The proper fix is to set those files in the docker container by upgrading the base dependency. Might be tricky.